kong/api/routes/clustering.lua

```diff
@@ -0,0 +1,18 @@
+local clustering = require("kong.clustering")
+local kong = kong
+
+
+return {
+  ["/clustering/status"] = {
+    GET = function(self, db, helpers)
+      if kong.configuration.role ~= "admin" then
+        return kong.response.exit(400, {
+          message = "this endpoint is only available when Kong is " ..
+                    "configured to run as Control Plane for the cluster"
+        })
+      end
+
+      return kong.response.exit(200, clustering.get_status())
```

Comment:
Should we have a proxy endpoint on Kong's Admin API for this endpoint as well?
Since Admin API is how most people are familiar with today and it is what will be used to configure Kong, it makes sense to have this endpoint to be present on the Admin API.

Reply:
Not sure if I got this, this API is currently served though CP Admin API. Is there something I missed here?

Reply:
Right. CP Admin API has only one endpoint right now (this one) but for users who start moving towards this the new deployment model, it might help to have this endpoint also be present on the traditional Admin API of Kong.